src/Components/ArtistName/ArtistName.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styling from './ArtistName.module.css';
+
+const ArtistName = ({ name, index, totalArtists }) => {
+    const generateDelimiter = (index, length) => {
+        if (index < length - 1) {
+            return " & ";
+        }
+        return "";
+    };
```

Comment:
I found your solution very nice! You can make it shorter by writing like this:
line 15
`{index<totalArtist -1? "&": ""}`
->You might need to write (index<totalArtist -1)